website/docs/components/accordion/index.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Accordion
-description: An accordion is a vertically stacked list of container-like toggles that reveal or hide associated sections of content.
-caption: A list of toggles that reveal or hide associated sections of content.
+description: (CV -) An accordion allow users to expand or collapse multiple sections of content in a stack. (KB -) An accordion is a vertically stacked list of container-like toggles that reveal or hide associated sections of content.
+caption: (CV -) Allow users to display multiple sections of content in a stack. (KB -) A list of toggles that reveal or hide associated sections of content.
```

Comment:
I changed this branch to point to the main documentation branch so it can be merged there first before being merged together with the ember component code.

There were a bunch of conflicts I fixed. For this section I kept both @cveigt and my text descriptions so they wouldn't be lost.

@cveigt You can edit the text here to keep the versions you like. (Let me know if this is too confusing and you need help.)

Reply:
@cveigt Could you edit these sections as to how you see fit? Right now they include both your and my content.

Reply:
No problem. I've resolved the conflicts and updated the content. Thanks for letting me know.